### PR TITLE
Use Adaptive Cards in MS Teams integration

### PR DESCRIPTION
**Pull request by santihernandezc**
This PR updates the MS Teams integration to use Adaptive Cards.
The code is mostly taken from https://github.com/grafana/alerting/blob/main/receivers/teams/teams.go

### CODE REVIEW

File: notify/msteams/msteams.go [hunk 1]
```diff
@@ -35,9 +35,15 @@ import (
 )
 
 const (
-	colorRed   = "8C1A1A"
-	colorGreen = "2DC72D"
-	colorGrey  = "808080"
+	ColorAlertFiring   = "#D63232"
```
Comment (grobinson-grafana) [inline notify/msteams/msteams.go:38]: I think `ColorAlertFiring` and `ColorAlertResolved` can be deleted as not used?

Comment (santihernandezc) [reply notify/msteams/msteams.go:38]: Ah! Good catch, I removed the need for these constants right before submitting the code but forgot to remove the constants 😅

Comment (alexander-akhmetov) [reply notify/msteams/msteams.go]: Removed: https://github.com/grafana/prometheus-alertmanager/pull/90/commits/bf5d7a3cf1725ce45e845fd4794b07fa006d2f3e

**Comment by alexander-akhmetov** [comment]
Updated and added a few tests, please have a look :)

**Review by titolins — APPROVED** [review]
🚀

**Review by grobinson-grafana — COMMENTED** [review]
Thanks for the tests, look good to me!

**Pull request merged**